release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -100,6 +100,18 @@ In {product-title} 4.12, the following improvements have been made in the *Helm*
 * With this update, you can view the list of the existing Helm chart repositories with their scope in the *Repositories* page.
 * With this update, you can view the newly created Helm release in the *Helm Releases* page.
 
+[id="ocp-4-12-enabling-disabling-developer-catalog"]
+==== Enabling or Disabling Developer catalog or one of it's sub-catalogs
+
+* With this update, the following options can be enabled or disabled in the Developer catalog by using either the form view or the YAML view:
+
+* Builder images
+* Devfiles
+* Helm Charts
+* Operator Backed
+* Samples
+* Templates
```

Comment:
And a few more sub-catalog types show up in the developer catalog when certain operators are installed. For ex: 'Event Sinks' and 'Event Sources' show up when the Serverless operator is installed.  I think we should add those as well

Reply:
As discussed in Slack, I have included the additional options you suggested in the content (https://github.com/openshift/openshift-docs/pull/52218) and I am rewording the text a bit as per the suggestion of a peer in the doc team. Kindly view this change and provide your feedback, if any.

Reply:
Fixed.